### ISSUE TITLE
Error Uploading events (Not able to see Track, Screen, Identity on Segment Dashboard.)

### ISSUE THREAD

**Issue by Jigar-zest** [segmentio/analytics-kotlin#117]
**Describe the bug**
I was working wth the Java version of the Plugin then I wanted to Move to Kotlin-analytics So I have updated to **1.8.0** After updating and making the necessary changes. Now I am not getting a single event in the Segment Dashboard.

`Note-: I also tried downloading a sample app and adding my private key still not working`

**To Reproduce**
Steps to reproduce the behaviour:
1. Download the sample
2. Add your private key and run

**Expected behaviour**
All events should go to the Segment dashboard.

**Screenshots**

**Platform (please complete the following information):**
 - Library Version in use: 1.8.0
 - Platform being tested:  Android
 - Integrations in use: [Firebase]
 
**Segment Initialization**
```
 companion object {
      lateinit var analytics: Analytics
    }
    
 Analytics.debugLogsEnabled = true
        analytics = Analytics(AppEnvironment.segmentAnalyticsKey, applicationContext) {
            this.collectDeviceId = true
            this.trackApplicationLifecycleEvents = true
            this.trackDeepLinks = true
            this.flushAt = 1
            this.flushInterval = 0
        }
```
**Additional context**
I am not getting any errors in the log console. Please let me know if you need any more information.

**Comment by Jigar-Iamplus** [comment]
Hey @bsneed 
Could take look at it. and let me know if I am missing something here.

**Comment by Jigar-zest** [comment]
Hey @bsneed 
Could you take a look and the above and let me know if I am missing something?

**Comment by wenxi-zeng** [comment]
hey @Jigar-zest. we're looking into this issue. do you mind to post your complete Analytics initialization block here, so we know exactly what's going on? 
while we're investigating, can you trying the following on your end:
1. remove `Analytics.debugLogsEnabled = true`
2. double check the private key you mentioned is the `writeKey` on your source Settings -> API Keys -> Write Key

**Comment by didiergarcia** [comment]
Hi @Jigar-zest I was able to confirm that the sample app works using a new key I just created so I don't think the SDK is the issue.

Some suggestions:

- Make sure your device emulator is connected to the internet.
- Double check your key and make sure you're not missing some part of due to a copy/paste issue.
- In the Sample app make sure you're updating the top-level `build.gradle` to set your `writeKey`
